### ISSUE TITLE
Remove credentials from repo

### ISSUE THREAD

**Issue by julia326** [COVID19Tracking/covid-publishing-api#12]
Right now we're storing encrypted credentials to access a public Google spreadsheet. We should either move those to GitHub secrets, or find another way to access the spreadsheet data.

**Comment by joshuaellinger** [comment]
I am blocked on this until you approve my Pull Request.  I have the code finished but I'm waiting for you to approve the other PR.

**Comment by joshuaellinger** [comment]
It looks like this got pulled in when I merged the branch due to a misunderstanding on my part about how feature branches work (b/c I've never used them in the past).

What happened is:
   1. I submitted the PR a day ago.
   2. Julia reviewed it but did not approve it.
   3. Last night, I pushed this change to the branch
   4. Julia approved it this morning.
   5. All the changes went into production.
